osc-server/src/main/java/org/osc/core/server/Server.java

```diff
@@ -118,7 +118,7 @@
 
     // static to avoid many trivial references
     private static Integer apiPort = DEFAULT_API_PORT;
-    private static volatile boolean inMaintenance = false;
+    private volatile boolean inMaintenance = false;
```

Comment:
@emanoelxavier you are correct that this is now a per-instance variable. As the server component is a singleton there will only be one instance at a time, but this will be re-created if any of its dependencies become unavailable. This would reset the server maintenance status, but it would also restart the whole server